charts/merlin/templates/_helpers.tpl

```diff
@@ -240,3 +240,12 @@ MLflow Postgres related
 {{- printf ""}}
 {{- end }}
 {{- end }}
+
+{{- define "authorization.server.url" -}}
```

Comment:
maybe put it into the commons chart?

Reply:
it's actually using commons helper functions inside. I think it's ok here considering we've reduced the duplicity of code using common functions.